Remove the "license" key from pyproject.toml, because the setuptools version used in SlapOS cannot read the SPDX license string of PEP 639.

--- msgpack-1.2.3.orig/pyproject.toml	2026-09-29 10:41:54.464795800 +0900
+++ msgpack-1.2.3/pyproject.toml	2026-10-18 10:45:48.146005575 +0900
@@ -5,7 +5,6 @@
 [project]
 name = "msgpack"
 dynamic = ["version"]
-license = "Apache-2.0"
 authors = [{name="Inada Naoki", email="songofacandy@gmail.com"}]
 description = "MessagePack serializer"
 readme = "README.md"
